registry/sql-registry/registry/models.py

```diff
@@ -432,9 +465,9 @@ def __init__(self,
         self._source = None
```

Comment:
Source and features are used for relation, but shall we add source id param to create_project_anchor_feature? 
otherwise there is no way for anchor features to see "source" of the anchor they belong to.

Reply:
The source id is set elsewhere.
XYZAttributes classes are **not** supposed to be created manually, it's a part of `Entity` object and should be deserialized from JSON/dict or converted from XYZDef, former is to read from database, latter is to handle user creation requests.
If you want to create AnchorAttributes, first you should have a AnchorDef object, which is provided by user in POST req, then call its `to_attr` method to create the blanked AnchorAttributes, I cannot fill `source` here because there is only an UUID in AnchorDef, but in AnchorAttributes we need a `EntityRef` which also includes the qualified name, this name can only be found when you have the whole registry data. So you need to manually fetch this source Entity object and set it as the property `source` of AnchorAttributes object.
Because we don't support updating right now, so the `_features` property is always empty when an anchor just got created, you can just leave it as is.